src/docfx/lib/JsonUtility.cs

```diff
@@ -145,7 +145,7 @@ public static string Serialize(object graph, Formatting formatting = Formatting.
         }
 
         /// <summary>
-        /// De-serialize a string to an object
+        /// De-serialize a user input string to an object, return error list at the same time
         /// </summary>
         public static (List<Error> errors, T model) Deserialize<T>(string json)
         {
```

Comment:
better name the method with its functions, not usage...

Reply:
Whatever name you choose, keep it in sync with `YamlUtility`